varnishgather

```diff
@@ -670,14 +685,27 @@ run lsblk
 #	run blockdev --getbsz /dev/$d
 #done
 
-
-for a in /var/log/messages /var/log/syslog; do
-	if [ -r "$a" ]; then
-		run egrep -i "(broadcaster|varnish|vha-agent|hitch|vac|rc.local|varnish-controller|vcs)" "$a"
-	else
-		incr
-	fi
-done
+capturelog /var/log/messages
+capturelog /var/log/syslog
+capturelog /var/log/debug
+capturelog /var/log/kern.log
+capturelog /var/log/dpkg.log
+capturelog /var/log/yum.log
+capturelog /var/log/dnf.log
+capturelog /var/log/user.log
+
+run journalctl -u varnish.service
```

Comment:
Could perhaps be useful to add a --since=yesterday or similar.

Reply:
Woulld `--since=-604800` be better? That's one week of journal.

Reply:
How about `--boot --lines=25000`? 

That will capture up to 25000 lines since the last time the host was rebooted.
I don't think we'll need more, and it's rarely that many since boot anyways.
If it has a lot of messages, it is usually the same messages over and over and it compresses well.

Varnish log messages seems to be around ~110 characters long, so 25000 lines will cap the log at around 2.75 MB uncompressed.

Reply:
yes to any of these, but it would also be nice to get the first few lines following the boot. If something misfires, it would be nice to see it.

but don't let me derail that discussion. I think the time widow makes a bit more sense that the number of lines, notably if some service is very chatty, but that's not a hard stance.

Reply:
Yes, I agree I would prefer to keep the time window.
I think if the service is too verbose it should be fixed there instead.
It is common for services to have a verbose level, if the user set this level too high then they get more logs :)

Reply:
cool, let's merge?

Reply:
I have updated the PR:
- Removed capture of compressed logs (I don't think it is needed anymore when we got journal logs). We could maybe also remove the capture of syslog and messages at some point.
- Capture a few more services.

Please have another look.
Meanwhile, I will have someone test this on a busy server.